src/trix/elements/trix_editor_element.js

```diff
@@ -238,6 +233,18 @@ export default class TrixEditorElement extends HTMLElement {
     this.editor?.loadHTML(this.defaultValue)
   }
 
+  get disabled() {
+    return this.inputElement.disabled
+  }
+
+  set disabled(value) {
+    this.toggleAttribute("disabled")
+  }
+
+  get type() {
```

Comment:
Is this just for testing purposes? Could you replace with just `instanceof` in tests?

Reply:
It isn't solely for testing. I've added the property because it's mentioned in the [Defining a form-associated custom element](https://web.dev/articles/more-capable-form-controls#form-associated_custom_elements) section of the _More capable form controls_ article.

Having said that, it's implemented using [Element.localName](https://developer.mozilla.org/en-US/docs/Web/API/Element/localName) (and not hard-coded like I've done here). It also doesn't seem to be mentioned by the MDN documentation for [ElementInternals](https://developer.mozilla.org/en-US/docs/Web/API/ElementInternals), so it isn't clear how necessary it is to start.